index.html

```diff
@@ -1978,8 +1978,8 @@ <h3>
               URL</var> as the base URL. If the result is failure, <a>issue a
               developer warning</a> and [=iteration/continue=].
               </li>
-              <li>If <var>shortcut</var>["url"] is not <a>within scope</a> of
-              <var>manifest URL</var>, <a>issue a developer warning</a> and
+              <li>If <var>shortcut</var>["url"] is not [=URL/within scope=]
```

Comment:
There seem to be changes leaking both ways between this and #886. I think you should not change this here since you have a separate PR to fix this bug.

Reply:
D'oh... seems wires got crossed by me jumping back and forth between the two branches... will split this up again...